packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -264,10 +265,17 @@ function useMediaUploadQueue() {
           // TODO: Only transcode & optimize video if needed (criteria TBD).
           // Probably need to use FFmpeg first to get more information (dimensions, fps, etc.)
           if (isTranscodingEnabled && canTranscodeFile(file)) {
+            const ext = getFileExtFromUrl(resource.src);
```

Comment:
Why do we need `getFileExtFromUrl` when we can just derive the file extension from the mimeType? 🤔

Reply:
Nope, an m4v and mp4 both have the mime type of video/mp4. 

This is also about just keeping the file name, if you named it wrong, as say .mov, this way we keep that.